src/tests/class_tests/openms/source/MRMIonSeries_test.cpp

```diff
@@ -169,9 +202,10 @@ END_SECTION
 START_SECTION((void MRMIonSeries::annotateTransitionCV(ReactionMonitoringTransition & tr, String annotation)))
 {
   MRMIonSeries mrmis;
-  ReactionMonitoringTransition tr, tr2;
+  ReactionMonitoringTransition tr, tr2, tr3;
 
   mrmis.annotateTransitionCV(tr, "y5^3");
+  mrmis.annotateTransitionCV(tr2, "y5-H2O1^3");
   mrmis.annotateTransitionCV(tr2, "y5-18^3");
```

Comment:
same comment as above, it would be nice to have this somewhere documented what these strings mean and what OpenMS understands and what it does not